apps/frontend/src/components/UserFeedbackModal/UserFeedbackModal.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
You shouldn't (and won't need to) import `React` itself anywhere. Instead, only import the definitions you need.